app/ios/iosinterface.mm

```diff
@@ -133,6 +217,11 @@ -( void )showImagePicker:( int )sourceType : ( IOSImagePicker * )handler
         QImage image = fromUIImage( chosenImage );
         QVariantMap data;
         data["image"] = image;
+        if ( !isCameraPhoto )
+        {
+          QString imagePathData( [imagePath UTF8String] );
+          data["imagePath"] = imagePathData;
+        }
```

Comment:
?

Reply:
There is now extra handling for gallery image - the image file is copied during this action.
For camera image, its kept as it was - image data are sent as response data and saved in the callback.